help/en/releasenotes/current-draft-note.shtml

```diff
@@ -402,7 +402,7 @@
     <h3>Layout Editor</h3>
         <a id="LE" name="LE"></a>
         <ul>
-	        <li></li>
+	        <li>This a problem that was causing Memory and Block contents to be slow to update.</li>
```

Comment:
No need to change, but the sentence might be better if "This" is changed to "Fixed".
`Fixed a problem that was causing Memory and Block contents to be slow to update.`